src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -13,28 +13,29 @@
     + 'You can still respond until they do.',
     { claimantName: claimantName } ) }}
   {% elseif status === ClaimStatus.CCJ_REQUESTED %}
-    {{ t('{{ claimantName }} requested a County Court Judgment against you on {{ ccjRequestedAt }}. '
-      + 'We will contact you both within 5 working days.',
-      { claimantName: claimantName, ccjRequestedAt: claim.countyCourtJudgmentRequestedAt | date }) }}
+    {{ t('The claimant has requested a County Court Judgement (CCJ) against you on {{ ccjRequestedAt }}.',
```

Comment:
It would be good to refactor `caseStatus.njk` as we discussed yesterday.

Reply:
I think that should be in a separate ticket. Otherwise the devs are changing the AC as we see fit and thats a slippery slope we addressed in the retro. 
It would also speak up how long it takes to get this story finished